src/Enums/PaymentType.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace RetailCosmos\IoiCityMallSalesFile\Enums;
+
+enum PaymentType: string
+{
+    case CASH = 'cash';
+
+    case TNG = 'tng';
+
+    case VISA = 'visa';
+
+    case MASTERCARD = 'mastercard';
+
+    case AMEX = 'amex';
+
+    case VOUCHER = 'voucher';
+
+    case OTHERS = 'others';
+
+    public static function names(): array
+    {
+        return array_column(self::cases(), 'name');
+    }
```

Comment:
Are we using this?

Reply:
Nope. But added, thought it could be useful. being used in `array()`

Reply:
We can remove both for now and add when needed please